test/edu-benefits/1990-rjsf/config/dependents.unit.spec.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-dom/test-utils';
+import { getFormDOM } from '../../../util/schemaform-utils';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990-rjsf/config/form.js';
+
+describe('Edu 1990 dependents', () => {
```

Comment:
I also like testing the `depends` functions for pages with them directly. You can get it from the form config and call it whatever data makes sense.